src/it/projects/profile-with-deps-inherit-parent-depMgmt-flatten-dep-all-inactive/pom.xml

```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.codehaus.mojo.flatten.its</groupId>
+    <artifactId>profile-with-deps-inherit-parent-depMgmt-flatten-dep-all-inactive</artifactId>
```

Comment:
I'm confused by the naming of this test.. why is this considered inactive?

Reply:
We will not specify the profiles we want to activate when running the test. I can add <activeByDefault>false</activeByDefault> to make it clearer

Reply:
SG

Reply:
Already update